description: manpage updates/fixes
author: Daniel Echeverry <epsilon77@gmail.com>
forwarded: http://code.google.com/p/chromium/issues/detail?id=73591
# This patch also includes ungoogled-chromium-specific information.

--- a/chrome/app/resources/manpage.1.in
+++ b/chrome/app/resources/manpage.1.in
@@ -5,11 +5,17 @@
 .SH NAME
 @@PACKAGE@@ \- the web browser from Google
 
+This is a build of ungoogled\-chromium
+
 .SH SYNOPSIS
 .B @@PACKAGE@@
 [\fIOPTION\fR] [\fIPATH\fR|\fIURL\fR]
 
 .SH DESCRIPTION
+ungoogled\-chromium is a set of modifications to @@MENUNAME@@ for removing Google integration and enhancing privacy, control, and transparency. See ungoogled\-chromium's page and README for more details:
+.IP
+<https://github.com/Eloston/ungoogled\-chromium>
+.PP
 See the Google Chrome help center for help on using the browser.
 .IP
 <https://support.google.com/chrome/>
@@ -17,9 +23,28 @@ See the Google Chrome help center for he
 This manpage only describes invocation, environment, and arguments.
 
 .SH OPTIONS
+ungoogled\-chromium introduces a few new command-line flags. Please consult its README for more details.
+.PP
 @@MENUNAME@@ has hundreds of undocumented command-line flags that are added
 and removed at the whim of the developers.  Here, we document relatively
 stable flags.
+
+.TP
+\fB\-h \fRor\fB \-\-help\fR
+Show help output.
+
+.TP
+\fB\-g \fRor\fB \-\-debug\fR
+Start a debugging session within /usr/bin/gdb.
+
+.TP
+\fB\-\-temp-profile\fR
+Use a throw-away/temporary profile for this session.  This creates an entirely new user profile temporarily.  It is not the same as incognito mode.
+
+.TP
+\fB\-\-enable-remote-extensions\fR
+Allow installation and updates of remote extensions.
+
 .TP
 \fB\-\-user\-data\-dir\fR=\fIDIR\fR
 Specifies the directory that user data (your "profile") is kept in.
@@ -114,6 +139,7 @@ as
 See the GTK documentation for more:
 .IP
 <http://library.gnome.org/devel/gtk/stable/gtk-running.html>
+.IP
 <http://library.gnome.org/devel/gtk/stable/gtk-x11.html>
 
 .SH ENVIRONMENT
@@ -159,12 +185,29 @@ Default directory for cache data.  (Why?
 <http://standards.freedesktop.org/basedir-spec/latest/> .)
 
 .SH BUGS
-Bug tracker:
+ungoogled\-chromium bug tracker:
+.IP
+https://github.com/Eloston/ungoogled\-chromium/issues
+.PP
+Please submit bug reports to ungoogled\-chromium first, unless you are certain
+that it is not specific to ungoogled\-chromium.
+.PP
+Debian bug tracker:
+.IP
+http://bugs.debian.org/chromium-browser
+.PP
+Upstream bug tracker:
 .IP
 http://code.google.com/p/chromium/issues/list
 .PP
 Be sure to do your search within "All Issues" before reporting bugs,
 and be sure to pick the "Defect on Linux" template when filing a new one.
+.PP
+This build derives from Debian's Chromium. If your bug is specific to Debian,
+please submit it only to the Debian bug tracker.
+.IP
+http://www.debian.org/Bugs/Reporting
 
 .SH AUTHOR
 The Chromium team \- <http://www.chromium.org>
+ungoogled-chromium \- <https://github.com/Eloston/ungoogled-chromium>
